### PR TITLE
Add glossary

### PR DESCRIPTION
**Pull request by SamGuay**

### CODE REVIEW

File: content/glossary/_index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#   3 = Card
+#   4 = Citation
+view: 1
+
+# Optional header image (relative to `static/media/` folder).
+header:
+  caption: ""
+  image: "resources.jpg"
+---
+
```
Comment (SamGuay) [inline content/glossary/_index.md:16]:
```
{{% callout note %}}
A Markdown callout is useful for displaying notices, hints, or definitions to your readers.
{{% /callout %}}
```
That's what you want to use @flavioazevedo

Comment (flavioazevedo) [reply content/glossary/_index.md:16]: Thanks!

Comment (flavioazevedo) [reply content/glossary/_index.md:16]:
```suggestion
{{% callout note %}}
This is a working in progress version of the Glossary which are (probably) filled with mistakes. This uses an older, non-reviewed version of the Glosssary. We are trying to see whats the best way to show information and terms.
{{% /callout %}}

```

**Review by flavioazevedo — APPROVED** [review]
Thank you Sam

**Pull request merged**